packages/devtools_shared/lib/src/memory/event_sample.dart

```diff
@@ -163,12 +163,12 @@ class EventSample {
     this.extensionEvents,
   );
 
-  EventSample.gcEvent(this.timestamp, {ExtensionEvents? events})
+  EventSample.gcEvent(this.timestamp, {ExtensionEvents? theEvents})
```

Comment:
rename to events. The in scope var should be used.

Reply:
Then it will conflict with the field name

Reply:
you can use `this.events = events`